data-esnext.js

```diff
@@ -569,10 +569,13 @@ exports.tests = [
       */},
       res: {
         ie11: false,
+        firefox60: false,
+        firefox68: false,
         firefox75: false,
         firefox81: {
           val: 'flagged',
-          note_id: 'ff-private-instance-methods-and-accessors'
+          note_id: 'ff-private-instance-methods-and-accessors',
+          note_html: 'The feature have to be enabled via <code>javascript.options.experimental.private_fields</code> and <code>javascript.options.experimental.private_methods</code> settings under <code>about:config</code>.'
```

Comment:
```suggestion
          note_html: 'The feature has to be enabled via <code>javascript.options.experimental.private_fields</code> and <code>javascript.options.experimental.private_methods</code> settings under <code>about:config</code>.'
```

Reply:
It was already like this, I just moved the note.  But I fixed it anyway.

Reply:
for sure, which is why i stamped anyways :-) <3 for fixing it tho!